onnxruntime/contrib_ops/cpu/transformers/logits_processor.cc

```diff
@@ -9,6 +9,8 @@ namespace onnxruntime {
 namespace contrib {
 namespace transformers {
 
+static int beam_search_iteration;
```

Comment:
It is better not be stored as a global variable. (Consider that two nodes executed at the same time, that means this code is not thread safe).

Could we pass the parameter in Process function instead?

Reply:
https://msdata.visualstudio.com/Vienna/_sprints/backlog/ONNX%20Inference/Vienna/Nickel?workitem=1606691